app/src/webkit/java/org/mozilla/focus/webkit/WebkitView.java

```diff
@@ -262,6 +262,12 @@ public boolean onShowFileChooser(WebView webView,
 
             return callback.onShowFileChooser(webView, filePathCallback, fileChooserParams);
         }
+
+        @Override
+        public void onReceivedTitle(WebView view, String title) {
+            super.onReceivedTitle(view, title);
+            callback.onReceivedTitle(view,title);
```

Comment:
We'll probably need callback null check. I'm seeing crashes here when topsite -> home -> topsite -> home -> topsite.